### PR TITLE
Adds Clifton Strengths test link to profile if user has no strengths

### LINKED ISSUE #1319: My Profile: When user has no CliftonStrengths Themes, show url to take the assessment
And, Display this in their profile in some manner to help them start the assessment.

### PR DESCRIPTION
**Pull request by sofistonehouse**
When the user has no Clifton Strengths, this will now display a link to the site where they can take the Clifton Strengths test. 
fixes #1319

### CODE REVIEW

File: src/components/Profile/components/PersonalInfoList/index.jsx [hunk 1]
```diff
@@ -261,11 +262,11 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
...
-  const cliftonStrengths =
-    profile.CliftonStrengths && (myProf || !profile.CliftonStrengths.Private) ? (
-      <ProfileInfoListItem
-        title="Clifton Strengths:"
-        contentText={
+  const cliftonStrengths = (
+    <ProfileInfoListItem
+      title="Clifton Strengths:"
+      contentText={
+        profile.CliftonStrengths && (myProf || !profile.CliftonStrengths.Private) ? (
```
Comment (bennettforkner) [inline src/components/Profile/components/PersonalInfoList/index.jsx:269]: If you use the logic above, then you can also simplify this line to only check if they have clifton strengths:
```suggestion
        profile.CliftonStrengths ? (
```

**Review by bennettforkner — CHANGES_REQUESTED** [review]
Hi Sofi, this is looking good so far. I think there are still a few logic holes that we are missing, so I'm requesting changes.

File: src/components/Profile/components/PersonalInfoList/index.jsx [hunk 2]
```diff
@@ -261,11 +262,11 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
       <ProfileInfoListItem title={'Graduation Year:'} contentText={profile.PreferredClassYear} />
     ) : null;
 
-  const cliftonStrengths =
-    profile.CliftonStrengths && (myProf || !profile.CliftonStrengths.Private) ? (
-      <ProfileInfoListItem
-        title="Clifton Strengths:"
-        contentText={
+  const cliftonStrengths = (
```
[the remark this answers is not in the document]
Comment (bennettforkner) [reply src/components/Profile/components/PersonalInfoList/index.jsx:265]: If you choose to implement this change, I might put the `const showCliftonStrengthsBlock = ...` somewhere near the top of the component.

File: .env.production [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Refers to a production environment, which may be either train or prod
 
 # @PROD
-# VITE_API_URL=https://360Api1.gordon.edu/
-# VITE_ANALYTICS_ID=UA-3559968-3
+ VITE_API_URL=https://360Api1.gordon.edu/
+ VITE_ANALYTICS_ID=UA-3559968-3
```
Comment (sofistonehouse) [inline .env.production:5]:
```suggestion
# VITE_API_URL=https://360Api1.gordon.edu/
# VITE_ANALYTICS_ID=UA-3559968-3
```

File: .env.production [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
 
 # @TRAIN
-VITE_API_URL=https://360ApiTrain.gordon.edu/
+# VITE_API_URL=https://360ApiTrain.gordon.edu/
```
Comment (sofistonehouse) [inline .env.production:8]:
```suggestion
 VITE_API_URL=https://360ApiTrain.gordon.edu/
```

File: src/components/Profile/components/PersonalInfoList/index.jsx [hunk 5]
```diff
@@ -26,6 +26,7 @@ import UpdatePhone from './components/UpdatePhoneDialog';
 import styles from './PersonalInfoList.module.css';
 import AlumniUpdateForm from './components/AlumniUpdateForm';
 import CliftonStrengthsService from 'services/cliftonStrengths';
+import { FaLessThanEqual } from 'react-icons/fa';
```
Comment (sofistonehouse) [inline src/components/Profile/components/PersonalInfoList/index.jsx:29]:
```suggestion
```

File: .env.production [hunk 6]
```diff
@@ -5,4 +5,4 @@
 # VITE_ANALYTICS_ID=UA-3559968-3
 
 # @TRAIN
-VITE_API_URL=https://360ApiTrain.gordon.edu/
+ VITE_API_URL=https://360ApiTrain.gordon.edu/
```
Comment (bennettforkner) [inline .env.production:8]:
```suggestion
VITE_API_URL=https://360ApiTrain.gordon.edu/
```

**Review by bennettforkner — APPROVED** [review]
LGTM! Feel free to merge into develop when you are ready.

**Review by bennettforkner — APPROVED** [review]
Looks good to me! Ready for merge into develop branch.

**Pull request merged**